.circleci/config.yml

```diff
@@ -26,17 +30,19 @@ aliases:
 defaults: &defaults
   working_directory: ~/buie
   docker:
-    - image: cypress/base:14.19.0
+    - image: cimg/node:18.16.0
+
+version: 2.1
 
-version: 2
 jobs:
   lint:
     <<: *defaults
     steps:
       - checkout
-      - restore-cache: *restore-yarn-cache
+#      - run: *chmod-working-dir
```

Comment:
commenting out caching related config for now, to separate node 18 bump, so that the issue for caching can be resolved after this PR goes through